ui/src/overview.tsx

```diff
@@ -147,18 +148,22 @@ export function Overview() {
             height: "auto",
           }}
         >
-          <DemographicCharts
-            cohort={backendCohort}
-            extraControls={
-              <Button
-                variant="outlined"
-                component={RouterLink}
-                to={absoluteCohortReviewListURL(params, cohort.id)}
-              >
-                Review individuals
-              </Button>
-            }
-          />
+          {underlay.uiConfiguration.featureConfig?.disableCohortReview ? (
```

Comment:
Put this condition inside the extraControls parameter so DemographicCharts isn't duplicated. Also, use undefined rather than null.